spec/rails_app/.env

```diff
@@ -0,0 +1 @@
+HEADLESS=true
```

Comment:
avoid creating and committing `.env` files, we'll need them for kamal later on and they shouldn't contain test stuff anyway

Reply:
done. Should we add .env.local/.env.test to the git ignore and use it there?